scripts/add-validator/README.md

```diff
@@ -0,0 +1,38 @@
+# Add Validator
+
+This script will help us in adding the validator node to blockchain programmaticaly. 
```

Comment:
This script will add a Validator Node to the existing Cere Network programmatically.

Reply:
updated.